src/serialchemy/polymorphic_serializer.py

```diff
@@ -45,9 +45,15 @@ def _get_sub_serializers(cls, declarative_class):
             sub_cls.get_identity(): sub_cls for sub_cls in cls.__subclasses__() if sub_cls.get_identity()
         }
 
+        def get_subclasses(declarative_class):
```

Comment:
Just add a small doc
""" Recursively finds all subclasses of the current class"""